oj.gemspec

```diff
@@ -20,15 +19,16 @@ Gem::Specification.new do |s|
   s.summary = "A fast JSON parser and serializer."
   s.description = "The fastest JSON parser and object serializer."
   s.licenses = ['MIT']
-  s.required_ruby_version = ">= 2.4"
+  s.required_ruby_version = ">= 2.5"
```

Comment:
Changed this to 2.5 to match was is tested in CI, although the version is EOL already

Reply:
Sounds good